{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"networkUnreachable"},{"kind":"text","text":": "},{"kind":"typeIdentifier","identifier":"doc:\/\/SwiftIO\/documentation\/SwiftIO\/Errno","preciseIdentifier":"s:7SwiftIO5ErrnoV","text":"Errno"},{"kind":"text","text":" { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["macOS"]}]},{"kind":"content","content":[{"anchor":"discussion","level":2,"type":"heading","text":"Discussion"},{"type":"paragraph","inlineContent":[{"type":"text","text":"A socket operation was attempted to an unreachable network."}]},{"type":"paragraph","inlineContent":[{"type":"text","text":"The corresponding C error is "},{"type":"codeVoice","code":"ENETUNREACH"},{"type":"text","text":"."}]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/swiftio\/errno\/networkunreachable"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/SwiftIO\/documentation\/SwiftIO\/Errno\/networkUnreachable","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Network is unreachable."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"networkUnreachable"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Errno","preciseIdentifier":"s:7SwiftIO5ErrnoV"}],"title":"networkUnreachable","roleHeading":"Type Property","role":"symbol","symbolKind":"property","externalID":"s:7SwiftIO5ErrnoV18networkUnreachableACvpZ","modules":[{"name":"SwiftIO"}]},"hierarchy":{"paths":[["doc:\/\/SwiftIO\/documentation\/SwiftIO","doc:\/\/SwiftIO\/documentation\/SwiftIO\/Errno"]]},"references":{"doc://SwiftIO/documentation/SwiftIO/Errno":{"role":"symbol","title":"Errno","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"Errno"}],"abstract":[{"type":"text","text":"The Errno struct lists all the possible errors."}],"identifier":"doc:\/\/SwiftIO\/documentation\/SwiftIO\/Errno","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"Errno"}],"url":"\/documentation\/swiftio\/errno"},"doc://SwiftIO/documentation/SwiftIO/Errno/networkUnreachable":{"role":"symbol","title":"networkUnreachable","fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"networkUnreachable"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Errno","preciseIdentifier":"s:7SwiftIO5ErrnoV"}],"abstract":[{"type":"text","text":"Network is unreachable."}],"identifier":"doc:\/\/SwiftIO\/documentation\/SwiftIO\/Errno\/networkUnreachable","kind":"symbol","type":"topic","url":"\/documentation\/swiftio\/errno\/networkunreachable"},"doc://SwiftIO/documentation/SwiftIO":{"role":"collection","title":"SwiftIO","abstract":[{"type":"text","text":"The SwiftIO library allows you to access and control the hardware in an easy way."}],"identifier":"doc:\/\/SwiftIO\/documentation\/SwiftIO","kind":"symbol","type":"topic","url":"\/documentation\/swiftio"}}}